app/src/main/java/com/example/candyshop/MainActivity.kt

```diff
@@ -3,44 +3,168 @@ package com.example.candyshop
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.tooling.preview.Preview
-import com.example.candyshop.ui.theme.CandyShopTheme
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavController
+import androidx.navigation.compose.rememberNavController
+import java.text.NumberFormat
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
-            CandyShopTheme {
-                // A surface container using the 'background' color from the theme
-                Surface(
-                    modifier = Modifier.fillMaxSize(),
-                    color = MaterialTheme.colorScheme.background
-                ) {
-                    Greeting("Android")
-                }
-            }
+            Navigation()
+//            CandyShopTheme {
+//                // A surface container using the 'background' color from the theme
+//                Surface(
+//                    modifier = Modifier.fillMaxSize(),
+//                    color = MaterialTheme.colorScheme.background
+//                ) {
+//                    CandyShopMain()
+//                }
+//            }
         }
     }
 }
 
 @Composable
-fun Greeting(name: String, modifier: Modifier = Modifier) {
-    Text(
-        text = "Hello $name!",
+fun CandyCard(name: String, imgId: Int, price: String, navController: NavController, modifier: Modifier = Modifier) {
+    Row(
         modifier = modifier
+            .padding(4.dp)
+            .fillMaxWidth()
+            .border(2.dp, Color(0xFFD3CEDF))
+            .size(width = 400.dp, height = 120.dp)
+            .padding(4.dp),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Image(
+            painter = painterResource(id = imgId),
+            contentDescription = null,
+            contentScale = ContentScale.Crop,
+            modifier = Modifier
+                .size(120.dp)
+                .border(2.dp, Color(0xFF9CB4CC), CircleShape)
+                .padding(2.dp)
+                .clip(CircleShape)
+        )
+        Column(
+            modifier = Modifier.padding(start = 16.dp)
+        ) {
+            Text(
+                text = name,
+                fontSize = 20.sp,
+                fontWeight = FontWeight.Bold,
+                color = Color(0xFF748DA6)
+            )
+            Text(
+                text = "price: $price",
+                color = Color(0xFF748DA6),
+                modifier = Modifier.padding(top = 6.dp, bottom = 14.dp)
+            )
+            Button(
+                onClick = {
```

Comment:
Maybe better idea would be to make whole CandyCard clickable? usually this is how this type of lists are working, you could also think about applying some ripple effect in case of click